vad_transcriber/wavTranscriber.py

```diff
@@ -88,7 +88,6 @@ def resolve_models(dirName):
 def vad_segment_generator(wavFile, aggressiveness):
     logging.debug("Caught the wav file @: %s" % (wavFile))
     audio, sample_rate, audio_length = wavSplit.read_wave(wavFile)
-    assert sample_rate == 16000, "Only 16000Hz input WAV files are supported for now!"
```

Comment:
Keep asserting but fetch the value from the model: https://deepspeech.readthedocs.io/en/v0.6.1/Python-API.html#native_client.python.Model.sampleRate